this is required to not install the development files of vendored libraries
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 98dd5dc..3ed3b7b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -377,7 +377,7 @@ if(USE_WIIUSE)
     else()
         # Fallback to built-in version
         if(WIIUSE_BUILD)
-            add_subdirectory("${PROJECT_SOURCE_DIR}/lib/wiiuse")
+            add_subdirectory("${PROJECT_SOURCE_DIR}/lib/wiiuse" EXCLUDE_FROM_ALL)
         endif()
         include_directories(BEFORE "${PROJECT_SOURCE_DIR}/lib/wiiuse/src/include")
         set(WIIUSE_LIBRARIES "wiiuse" "bluetooth")
@@ -411,7 +411,7 @@ if(USE_SYSTEM_ANGELSCRIPT)
             "-DUSE_SYSTEM_ANGELSCRIPT=0")
     endif()
 else()
-    add_subdirectory("${PROJECT_SOURCE_DIR}/lib/angelscript/projects/cmake")
+    add_subdirectory("${PROJECT_SOURCE_DIR}/lib/angelscript/projects/cmake" EXCLUDE_FROM_ALL)
     include_directories(BEFORE "${PROJECT_SOURCE_DIR}/lib/angelscript/include")
     set(Angelscript_LIBRARIES angelscript)
 endif()
